src/libraries/System.Private.CoreLib/src/System/Type.cs

```diff
@@ -583,5 +567,11 @@ public static Binder DefaultBinder
         public static readonly MemberFilter FilterNameIgnoreCase = (m, c) => FilterNameImpl(m, c!, StringComparison.OrdinalIgnoreCase);
 
         private const BindingFlags DefaultLookup = BindingFlags.Instance | BindingFlags.Static | BindingFlags.Public;
+        internal const DynamicallyAccessedMemberTypes MemberAnnotations = DynamicallyAccessedMemberTypes.PublicFields | DynamicallyAccessedMemberTypes.NonPublicFields |
```

Comment:
```suggestion
        internal const DynamicallyAccessedMemberTypes GetAllMembers = DynamicallyAccessedMemberTypes.PublicFields | DynamicallyAccessedMemberTypes.NonPublicFields |
```

With a comment that explains why/how this is different than DynamicallyAccessedMemberTypes.All